README.md

```diff
@@ -1 +1,186 @@
-# angular-ngroute-readme
\ No newline at end of file
+# What is ngRoute?
+
+## Overview
+
+Let's take a look at routing - this is the part of the application which spreads out your app into different pages.
+
+## Objectives
+
+- Describe ngRoute and ng-view
+- Write a route using ngRoute
+- Write a dynamic route using ngRoute
+- Integrate a Controller with ngRoute
+- Integrate a template with ngRoute
+
```

Comment:
Add a section here about routing in general and how front end frameworks handle routing. This section should include references to the following: 
1. Our URLs should represent the action a user is taking on the page. 
2. Because of this, a user should be able to send a URL to another user and have them see the exact same state. 
3. So far, none of the actions our users have taken have been reflected in the URL.
4. Angular provides us a way to reflect the state of the application in the URL - this not only makes for a better user experience, but it lets us split our application out into logical templates.